app/src/androidTest/java/ch/epfl/sdp/blindly/utils/CheckInternetTest.kt

```diff
@@ -0,0 +1,26 @@
+package ch.epfl.sdp.blindly.utils
+
+import android.content.Context
+import androidx.test.platform.app.InstrumentationRegistry
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+import org.junit.Before
+import org.junit.Test
+
+@HiltAndroidTest
+class CheckInternetTest {
+
+    lateinit var context: Context
+
+    @Before
+    fun setup() {
+        context = InstrumentationRegistry.getInstrumentation().context
+    }
+
+    @Test
+    fun isConnectedIsTrueIfConnected() {
+        val bool: Boolean = CheckInternet.internetIsConnected(context)
+        MatcherAssert.assertThat(bool, Matchers.equalTo(true))
```

Comment:
It could be nice to import directly MatcherAssert.assertThat so that is it less redundant (like in other tests)